scss/modules/_filters.scss

```diff
@@ -0,0 +1,164 @@
+// Filters Panel
+//
+// Markup:
+// <section class="layout--browse">
+// <div class="filters is-open">
+//   <button class="filters__toggle"></button>
+//   <div class="filters__hider">
+//     <h4 class="filter__header">Filters</h4>
+//     <div class="filter">
+//      Some filter
+//     </div>
+//     <div class="filter">
+//      Some filter
+//     </div>
+//   </div>
+//  </div>
+// </section>
+//
+// Styleguide modules.filters
+
+.filters {
+  @include clearfix();
+  @include transition(margin-left .2s);  
+  background: $neutral;
+  bottom: 0;
+  height: 0;
+  margin-right: 3%;
+  padding: 1rem;
+  position: absolute;
+  top: 0;
+  width: 100%;
+
+  @include media($lg) {
+    height: auto;
+    float: left;
+    margin-left: -20%;
+    padding: 2rem;
+    position: relative;
+    width: 20%;
+  }
+}
+
+.filters__hider {
+  height: 0;
+  overflow: hidden;
+
+  @include media($lg) {
+    overflow: visible;
+  }
+}
+
+
+.filters__toggle {
+  @include transform(translateX(-50%));
+  background-color: $neutral;
+  background-repeat: no-repeat;
+  background-position: center center;
+  left: 50%;
+  position: absolute;
+  z-index: 100;
+  top: 100%;
+
+  &:hover {
+
+  }
+
+  @include media($lg) {
+    @include transform(translateX(0));
+    background-image: url('../img/i-right-2--primary.png');
+    background-image: url('../img/i-right-2--primary.svg');
+    left: 100%;
+    top: 0;
+    width: 3rem;
+    height: 3rem;
+
+    &:hover, &:focus {
+
+    }
+  }
+}
+
+.filters__header {
+
+}
+
+.filters__subheader {
+  margin-top: 3rem;
+  padding-top: 2rem;
+  border-top: 2px solid $primary;
+
+  & + .filter {
+    padding-top: 0;
+  }
+}
+// Styles for when the filters are open
+.is-open {
```

Comment:
I feel like if you see `is-open` you might apply that to things that aren't filters. Is that OK?